File: launchdarkly/typescript-node/src/GetDeploymentsExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.InsightsDeploymentsBetaApi();
apiCaller.setApiKey(api.InsightsDeploymentsBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getDeployments(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  undefined, // applicationKey
  undefined, // limit
  undefined, // expand
  undefined, // from
  undefined, // to
  undefined, // after
  undefined, // before
  undefined, // kind
  undefined, // status
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling InsightsDeploymentsBetaApi#getDeployments:");
  console.log(error.body);
});

File: launchdarkly/ruby/src/GetDeploymentsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::InsightsDeploymentsBetaApi.new.get_deployments(
        nil, # project_key
        nil, # environment_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling InsightsDeploymentsBetaApi#get_deployments: #{e}"
end
